### PR TITLE
Document capitalization scheme for Linebender projects

### PR DESCRIPTION
**Pull request by PoignardAzur**

### CODE REVIEW

File: content/wiki/capitalization_scheme.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
++++
+title = "Capitalization scheme for Linebender projects"
++++
+
+All Linebender projects should follow this capitalization scheme:
+
+- The project's name should start with an uppercase letter when referring to the project itself.
+- The name should be all-lowercase where referring to the project as a Rust crate (for instance, in relation to crates.io) or as a Github repository.
+- In the latter two cases, it's encouraged to use backticks to put the project name in a codeblock, eg Vello vs `vello`.
```
Comment (DJMcNab) [inline content/wiki/capitalization_scheme.md:9]: I think we could have an example like [vello#1](https://github.com/linebender/vello/issues/1), which we would not expect to be in a codeblock

**Review by DJMcNab — APPROVED** [review]

**Comment by DJMcNab** [comment]
We do also need to check whether we want to retroactively apply this to druid. I think probably not, so that might need to be called out directly

**Comment by PoignardAzur** [comment]
I can either make a list or say something like "this doesn't apply to explicitly deprecated projects".

**Comment by DJMcNab** [comment]
I'd probably make a list.

Saying that, looking at the [Druid readme](https://github.com/linebender/druid), they already seem to mostly follow this scheme anyway

File: content/wiki/capitalization_scheme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
++++
+title = "Capitalization scheme for Linebender projects"
++++
+
+All Linebender projects should follow this capitalization scheme:
+
+- The project's name should start with an uppercase letter when referring to the project itself, eg Vello.
+- The name should be all-lowercase where referring to the project as a Rust crate (for instance, in relation to crates.io). In that case, it's encouraged to use backticks to put the project name in a codeblock, eg `vello`.
+- The name should be all-lowercase where referring to the project as a Github repository. In that case, it's encouraged to either use backticks or make it a link, eg [vello#1](https://github.com/linebender/vello/pull/1).
```
Comment (DJMcNab) [inline content/wiki/capitalization_scheme.md:9]: Reminder that we're aiming for one sentence per line now in the wiki

**Review by DJMcNab — APPROVED** [review]

File: content/wiki/capitalization_scheme.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+Feel free to correct any documentation you see to follow this scheme.
+
+(Projects that are discontinued or in maintenance mode like Piet, Druid or Xi-Editor don't need their documentation to follow this scheme.)
```
Comment (xStrom) [inline content/wiki/capitalization_scheme.md:13]: As Druid already follows this scheme I don't think it's needed to point it out as a project that doesn't need to follow it.

**Review by xStrom — APPROVED** [review]
> We do also need to check whether we want to retroactively apply this to druid. I think probably not, so that might need to be called out directly

This has been the policy for Druid since June of 2020, as was initially decided in [this Zulip thread](https://xi.zulipchat.com/#narrow/stream/147926-druid/topic/name.20style) and later clarified a bit in other threads. As such, there's nothing to retroactively apply.

---

Overall this PR looks like a good step towards documenting what our policies are. 🚀

File: content/wiki/capitalization_scheme.md (hunk 2 above)
Comment (PoignardAzur) [reply content/wiki/capitalization_scheme.md]: After trying it, I'm not really a fan of this formatting style. There's no obvious way to configure it in VsCode, it looks kind of off in the editor, and I doubt the version control benefits are that big.

**Pull request merged**